lib/class-wp-theme-json.php

```diff
@@ -271,25 +250,43 @@ class WP_Theme_JSON {
 		),
 	);
 
+	const ELEMENTS = array(
```

Comment:
Did you manage to extract the elements API to its own PR? did you abandon this idea?

Reply:
I was thinking of extracting some other PRs with client changes (mostly refactors, renames) but doing the theme.json format change in a single PR. Happy to re-consider if folks think that helps.